packages/ra-core/src/form/useWarnWhenUnsavedChanges.tsx

```diff
@@ -1,7 +1,7 @@
-import { useEffect, useRef } from 'react';
```

Comment:
did you backport the fix we recently did in master?

Reply:
Nope, should I?

Reply:
no, let's do it later